server/priv/gettext/marketing.pot

```diff
@@ -3,214 +3,213 @@
 ## "msgid"s here are often extracted from source code.
 ## Add new messages manually only if they're dynamic
 ## messages that can't be statically extracted.
-##
 ## Run "mix gettext.extract" to bring this file up to
 ## date. Leave "msgstr"s empty as changing them here has no
 ## effect: edit them in PO (.po) files instead.
 #
-#: lib/tuist_web/marketing/controllers/marketing_html/home.html.heex:418
+#: lib/tuist_web/marketing/controllers/marketing_controller.ex:122
```

Comment:
Need to figure out the right settings on Weblate to prevent `msgstr` from getting populated.